AutomaticStockTrader.Core/Alpaca/AlpacaCalendar.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Quartz.Impl.Calendar;
+using System;
+
+namespace AutomaticStockTrader.Core.Alpaca
+{
+    public class AlpacaCalendar : BaseCalendar
+    {
+        private readonly IAlpacaClient _alpacaClient;
+
+        public AlpacaCalendar(IAlpacaClient alpacaClient)
+        {
+            _alpacaClient = alpacaClient ?? throw new ArgumentNullException(nameof(alpacaClient));
+        }
+
+        public override DateTimeOffset GetNextIncludedTimeUtc(DateTimeOffset timeUtc)
+            => _alpacaClient.GetNextIncludedTimeUtc(timeUtc).Result;
```

Comment:
does this need to be sync?

Reply:
To override the base class, yeah it has to be synchronous.